src/script/interpreter.cpp

```diff
@@ -335,9 +335,70 @@ bool EvalScript(vector<vector<unsigned char> >& stack, const CScript& script, un
                 // Control
                 //
                 case OP_NOP:
-                break;
+                    break;
+
+                case OP_CHECKLOCKTIMEVERIFY:
+                {
```

Comment:
Can you move all this to an inline function outside of EvalScript?
It will be build equivalent but much more readable (at the very least for the indentation, but seeing the parameters of each opcode is interesting too).
At some point we should do something like #5153 (having a check-identical-build script would help making that trivial to review and non-risky in a completely obvious way), so we could at least start doing it with the new opcodes.